VAMobile/src/components/Nametag/Nametag.tsx

```diff
@@ -40,21 +45,42 @@ export const Nametag: FC = () => {
   }
 
   return (
-    <Box width="100%" backgroundColor="profileBanner" minHeight={85} display="flex" justifyContent="center" mb={theme.dimensions.standardMarginBetween} accessible={true}>
-      <Box py={accessToMilitaryInfo ? theme.dimensions.cardPadding : 0} display="flex" flexDirection="row">
-        {accessToMilitaryInfo && <Box pl={theme.dimensions.cardPadding}>{getBranchSeal()}</Box>}
-        <Box ml={20} flex={1}>
-          <TextView textTransform="capitalize" mb={theme.dimensions.textIconMargin} variant="BitterBoldHeading" color="primaryContrast">
-            {name()}
-          </TextView>
-          {accessToMilitaryInfo && (
-            <TextView textTransform="capitalize" variant="MobileBodyBold" color="primaryContrast">
-              {branch}
+    <Pressable onPress={navigateTo('VeteranStatus')} accessibilityRole={'button'} accessibilityLabel={name() + ' ' + branch + ' ' + t('veteranStatus.title')}>
```

Comment:
This could still just be pulled out and put in a function with navigateTo on the end?

Reply:
I had to do that in a few places in the past. Since navigateTo() returns a function, you have to call the function it returns. Here's an example:
```
const onPress = () => {
  logAnalyticsEvent(Events.vama_ddl_status_click())
  navigateTo('ClaimLettersScreen')()
}
```